mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                        not have an infoType, the DLP API applies the limit against all infoTypes that are found but not
+                        specified in another InfoTypeLimit.
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'name'
+                          required: true
+                          description: |
+                            Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names listed
+                            at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                        - !ruby/object:Api::Type::String
+                          name: 'version'
+                          description: |
+                            Version of the information type to use. By default, the version is set to stable
+                    - !ruby/object:Api::Type::Integer
+                      name: 'maxFindings'
+                      description: Max findings limit for the given infoType.
+                      required: true
+          - !ruby/object:Api::Type::Array
+            name: 'infoTypes'
+            description: |
+              Restricts what infoTypes to look for. The values must correspond to InfoType values returned by infoTypes.list
+              or listed at https://cloud.google.com/dlp/docs/infotypes-reference.
+
+              When no InfoTypes or CustomInfoTypes are specified in a request, the system may automatically choose what detectors to run.
+              By default this may be all types, but may change over time as detectors are updated.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'name'
+                  required: true
+                  description: |
+                    Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names listed
+                    at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                - !ruby/object:Api::Type::String
+                  name: 'version'
+                  description: |
+                    Version of the information type to use. By default, the version is set to stable
+          - !ruby/object:Api::Type::Array
+            name: 'ruleSet'
+            description: |
+              Set of rules to apply to the findings for this InspectConfig. Exclusion rules, contained in the set are executed in the end,
+              other rules are executed in the order they are specified for each info type.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Array
+                  name: 'infoTypes'
+                  required: true
```

Comment:
Just want to double check if this field is required.

Reply:
Checked this field, works fine without `required:true`. Removing that from attribute.